apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs

```diff
@@ -0,0 +1,73 @@
+use crate::infrastructure::olap::clickhouse::client::ClickHouseClient;
+use crate::infrastructure::olap::clickhouse::model::ClickHouseRecord;
+use std::time::Duration;
+
+use log::{debug, error};
+use std::sync::Arc;
+use tokio::sync::Mutex;
+use tokio::time;
+
+use super::config::ClickHouseConfig;
+
+const MAX_FLUSH_INTERVAL_SECONDS: u64 = 1;
+const MAX_BATCH_SIZE: usize = 1000;
```

Comment:
Should these values should be in the clickhouse connection config and available for population via environment variable overrides in cloud product?  Also we need to document the tradeoffs of tuning those values.

Reply:
That's a good question, from reading the docs it looks like the parameters from a clickhouse perspective are constrained, they are saying you should not make an insert more than 1 / sec and that the batches should be rather big. 

We could make it parametrizable with sane defaults but I wonder if a user would potentially just shoot themselves in a foot. Since we are trying to make it dead simple, unless absolutely necessary, I feel like we should hide parameters as much as possible.

I will leave it like this for now and we can change if we need to